build.sbt

```diff
@@ -105,18 +105,23 @@ lazy val validation = (project in file("validation"))
 
 lazy val panel = (project in file("panel"))
   .settings(hmdaBuildSettings: _*)
+  .settings(Revolver.settings:_*)
   .settings(
-    assemblyMergeStrategy in assembly := {
-      case "application.conf" => MergeStrategy.concat
-      case "META-INF/io.netty.versions.properties" => MergeStrategy.concat
-      case x =>
-        val oldStrategy = (assemblyMergeStrategy in assembly).value
-        oldStrategy(x)
-    },
-    libraryDependencies ++= akkaPersistenceDeps ++ httpDeps
+    Seq(
+      assemblyJarName in assembly := {s"${name.value}.jar"},
```

Comment:
What does the .jar name resolve to, and where will it be once it gets generated?

I ran `sbt clean assembly` in the project, then looked in `target/scala-2.12/` for the compiled .jar files. I only saw `hmda.jar`. I'm not sure if that means the panel loader .jar will live in a different target directory, or if it didn't get created correctly.